decidim-assemblies/spec/serializers/decidim/assemblies/assembly_serializer_spec.rb

```diff
@@ -0,0 +1,142 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim::Assemblies
+  describe AssemblySerializer do
+    let(:resource) { create(:assembly) }
+    let(:subject) { described_class.new(resource) }
+
+    describe "#serialize" do
+      it "includes the assembly data" do
+        serialized = subject.serialize
+
+        expect(serialized).to be_a(Hash)
+
+        expect(subject.serialize).to include(id: resource.id)
```

Comment:
why not use `serialized` instead?
```suggestion
        expect(serialized).to include(id: resource.id)
```
even more performant would be to convert the hash to string or to_json and compare it to an expected string